### PR TITLE
Create guide.json for guide

### PR DESCRIPTION
**Pull request by Specifer**
Implementing #518 guide.json for Dialog box of guide

### CODE REVIEW

**Comment by RussellLVP** [comment]
Is this an official list of regulations? I don't recall this being discussed.

Could we merge this with #547? The guide.json file and the JavaScript code need to be submitted together.

**Comment by Specifer** [comment]
Sure , the three requests are supposed to be merged .

**Comment by RussellLVP** [comment]
@LVPJoe please advice on this.

**Comment by BeanerLVP** [comment]
@RussellLVP  Specifer has made certain minor changes to the original content I posted in the ticket. I'm not sure why he would do that without discussing it with any of us. Kindly revert to the original content for now, and make further changes once this has been discussed & approved internally. 
I'll post a forum topic about it shortly.

Content that was originally proposed by me:

> As a temporary administrator, there are some key points you should keep in mind:
> 
> 1. Be nice, be respectful, and more importantly, stay calm in all situations.
> 2. Useful commands: /watch, /stopwatch, /p, /mute, /jail, /kick, /ban, /fix, /weapons, /vr, /cruise.
> 3. The default ban length is 3 days. Utilize verbal warnings, /mute, /jail, and /kick before banning people for minor offenses such as ship-nading, interior-nading, or swearing.
> 4. Always test for health hacks using /p handofgod and /fix. If they don't work on de-synced players, use /reconnect. DO NOT ban based on /p burn and /p nuke alone.
> 5. Recording & submitting video evidence of suspected hacking in case of regular/registered players is mandatory. The crew shall review it and take appropriate actions.
> 6. Use /crew [message] to send messages to the LVP Crew IRC channel, if you require any assistance.
> 
> Please visit https://administrator.guide/ for a general overview.
> Remember, this is only a game — don't take things personally. Stay unbiased at all times.
> 
> Thank you for volunteering to assist the Las Venturas Playground crew. We sincerely appreciate your efforts!

**Comment by RussellLVP** [comment]
Thank you for the input @BeanerLVP!

Do you feel there's consensus about your proposed text? It seems good to me, but I think others might want to read it before we instate it.

**Comment by Specifer** [comment]
I see that minor change is (Mandatory and /p Id health ) . Is it not of good ?Shall I change it to as what beaner asked ? (removing Mandatory and p health ).

File: data/commands/guide.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (Specifer) [inline data/commands/guide.json:1]: @RussellLVP , Reverted it as per as @BeanerLVP request (ie removed p health and mandatory) . Please comment on this .

Comment (BeanerLVP) [reply data/commands/guide.json:1]: Specifer, thank you for trying to contribute, but please leave this particular feature for us to deal with, as we're still in the process of discussing it internally.

**Comment by BeanerLVP** [comment]
Since nobody else has raised objections or suggested further edits, I think we can go ahead with this message:

```
As a temporary administrator, there are some key points you should keep in mind:

1. Be nice, be respectful, and more importantly, stay calm in all situations.
2. Useful commands: /watch, /stopwatch, /p, /mute, /jail, /kick, /ban, /fix, /weapons, /v, /vr, /flip, /cruise.
3. Utilize verbal warnings, /mute, /jail, and /kick more often over /ban. Remember, GTA is all about chaos and breaking the rules.
4. Avoid banning for minor offenses such as ship-nading, interior-nading, fake timeouts, command abusing in fights, and swearing when not done excessively.
5. Always test for health hacks using /p handofgod and /fix. If they don't work on de-synced players, use /reconnect. DO NOT ban based on the outcomes of /p burn and /p nuke alone.
6. Recording & submitting video evidence in case of suspected aimbot, no-recoil, dmg.cs, and other cheats of such nature is mandatory. The crew shall review it and take appropriate actions.
7. Use /crew [message] to send messages to the LVP Crew IRC channel if you require assistance. Always feel free to reach out to us.

Please visit https://administrator.guide/ for a general overview.
Remember, this is only a game — don't take things personally. Stay unbiased at all times.

Thank you for volunteering to assist the Las Venturas Playground crew. We sincerely appreciate your efforts!
```

**Comment by Specifer** [comment]
OK.. However let me test this in my local server one final time.

**Pull request closed without merging**